### PR TITLE
Math-Magicians - Components

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request:
- Created React Components to Build Calculator UI
- Styled Calculator UI to match guide

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```
Comment (t-yanick) [inline src/App.js]: - [x] As it is a [requirement](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/react_redux/math_magicians/2_Components.md) for this project " `Use class based components (All components should be class-based)` ", please kindly covert your `App.js` component which is currently a functional component to a class-based component. 

```javascript
class App extends Component {
  render() {

    return {
      <Calculator />
    };
  }
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import './Calculator.css';
+import CalcButton from './CalcButton';
+import Display from './Display';
+
+export default function Calculator() {
```
Comment (t-yanick) [inline src/components/Calculator.js]: - [x] As it is a [requirement](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/react_redux/math_magicians/2_Components.md) for this project " `Use class based components (All components should be class-based)` ", please kindly covert your `Calculator.js` component which is currently a functional component to a class-based component.  Some like so: 

import React, { Component } from 'react';

class Calculator extends Component {
  render() {

    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @sabetech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and  under the review._

#### I will leave these comments here since they concern files in your repository and your readme file on which you haven't made any changes for this `PR`.

- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Delete all the boilerplate from CRA (text, images, styles).` ", please checkout  your `public and src directories` and delete all files the are not related to this project. 
- [ ] Kindly update the `Getting Started` section of your readme file with explicit steps on how to clone and run your app locally as this will be of help to non technical users.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Whoistolu) [reply src/App.js]: - [x] [REQUIRED] As recommended by the previous reviewer, kindly ensure that the `App.js` component is class-based.

```
class App extends Component {
  render() {

    return {
      <Calculator />
    };
  }
}
```

File: src/components/Calculator.js (hunk 2 above)
Comment (Whoistolu) [reply src/components/Calculator.js]: - [ ] [REQUIRED] As recommended by the previous reviewer, kindly ensure that the `Calculator.js` component is class-based.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @sabetech 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (sabetech) [reply src/App.js]: Thanks for the comment. The components have been changed to class-based components as required

**Review by brytebee — APPROVED** [review]
Hi, @sabetech👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**